tests/Pyxsim/testers.py

```diff
@@ -0,0 +1,114 @@
+import re
```

Comment:
Needs a copyright comment and a comment referencing the license.  See the Source check failures in the Jenkins run.

Reply:
added